src/elements/category.js

```diff
@@ -0,0 +1,40 @@
+export default function(namespace) {
+  const ArrayElement = namespace.getElementClass('array');
+
+  class Category extends ArrayElement {
+    constructor() {
+      super(...arguments);
+      this.element = 'category';
+    }
+
+    get resourceGroups() {
+      return this.children((item) => item.classes.contains('resourceGroup'));
+    }
+
+    get dataStructures() {
+      return this.children((item) => item.classes.contains('dataStructures'));
+    }
+
+    get scenarios() {
+      return this.children((item) => item.classes.contains('scenario'));
+    }
+
+    get transitionGroups() {
+      return this.children((item) => item.classes.contains('transitions'));
+    }
+
+    get resources() {
+      return this.children((item) => item.element === 'resource');
+    }
+
+    get transitions() {
+      return this.children((item) => item.element === 'transition');
+    }
+
+    get copy() {
```

Comment:
I seems to have missed this earlier. I wonder why the 2 functions above are named `resources` and `transitions` while this one is named `copy`. Shouldn't this be named `copies`?

IMO: Maybe we should rename the other functions as `resource` and `transition`.